### PR TITLE
Add documentation for Kotlin Notebook Plugin usage and troubleshooting

### PR DESCRIPTION
**Pull request by ermolenkodev**
This update introduces new documentation sections covering the usage of the Kotlin Notebook Plugin and a troubleshooting guide.

### CODE REVIEW

**Review by koperagen — APPROVED** [review]
Very interesting content!

File: docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Usage with Kotlin Notebook Plugin
+
+The [Kotlin Notebook Plugin for IntelliJ IDEA](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook), combined with Kotlin DataFrame, offers powerful data analysis capabilities within an interactive environment. Here are key features:
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md:3]: the key features

File: docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### Visual Data Exploration
+**Page through your data**:
+The pagination feature lets you move through your data one page at a time, making it possible to view large datasets.
+
+**Sort by column with a single click**:
+You can sort any column with a click. This is a convenient alternative to using sortBy in separate cells.
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md:10]: I'd write sortBy like `` `sortBy` ``

File: docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+**Go straight to the data you need**:
+You can jump directly to a particular row or column if you want something specific.
+This makes working with large datasets more straightforward.
+<video src="https://github.com/user-attachments/assets/aeae1c79-9755-4558-bac4-420bf1331f39" controls></video>
+
+### Drill down into nested data
+When your data has multiple layers, like a table within a table, you can now click to view these details directly.
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md:18]: click on what? :) I'd specify "click on a cell containing a nested table" or something like that

File: docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+This makes it easy to go deeper into your data and then return to where you were.
+<video src="https://github.com/user-attachments/assets/ef9509be-e19b-469c-9bad-0ce81eec36b0" controls></video>
+
+### Visualize multiple tables via tabs
+You can open and visualize multiple tables in separate tabs.
+This feature is tailored for those who need to compare, contrast, or simply monitor different datasets simultaneously.
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md:24]: tailored to

File: docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+<video src="https://github.com/user-attachments/assets/51b7a6e3-0187-49b3-bf5e-0c4d60f8b769" controls></video>
+
+
+### Exporting to files
+
+You can export data directly from the DataFrame into various file formats. This simplifies sharing and further analysis.
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md:30]: *dataframe, you refer to the concept I think

File: docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+The component supports exporting data to JSON for web applications, CSV for spreadsheet tools, and XML for data interchange.
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md:31]: users don't know what a "component" is in this context. I'd say "The tables support..."

**Comment by ermolenkodev** [comment]
@Jolanrensen Would you like to take a look?

**Review by Jolanrensen — APPROVED** [review]
all good :)

**Pull request merged**